### DISCUSSION TITLE
前 100 个发帖人可以获得纪念 NFT（纯水贴除外）

### DISCUSSION THREAD

**Discussion by luckyyang** [rebase-network/hello-world#8] [Announcements]
在论坛发帖的前 100 个人可以获得纪念版 NFT，每人一个。

参与条件:
- 在论坛发帖
- 在这个帖子下面留言，附上自己的帖子链接和 CKB 钱包，钱包地址可以使用 https://flashsigner.com/ 注册

一些无意义的帖子不会计算在这次活动中，最终解释权由 Rebase 社区所有。

**Comment by eliezhao** [comment]
Link : https://github.com/rebase-network/hello-world/discussions/18    
wallet : ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqfdsuzq2pueg0pvdfr7h2mn7jadf5s5zxs4a2lhs

**Comment by zhengshuai666** [comment]
https://github.com/rebase-network/hello-world/discussions/20
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqdhh8l83wzz062g4epk704tk498c7rgycqwya9mc

**Comment by DracoYan-111** [comment]
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqthhr5vj5vev25wygrkexsyzfg9mnstyls0jgzdl

**Comment by carrot93** [comment]
[支持实现 nft database](https://github.com/rebase-network/hello-world/discussions/26)
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqtrj80t8daee2l0v7g4z3heavpt9red5asfk9h47

**Comment by 1720433410** [comment]
https://github.com/rebase-network/hello-world/discussions/43
0x93373ec820d68A9dD3E52122E083C9fB990Fef2c

**Comment by gaoshanliushui662** [comment]
https://github.com/rebase-network/hello-world/discussions/45
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsq2gzvs40z2t56gd3z9c4fxh9cnsqme9n9sfjwavn

**Comment by BradMoonUESTC** [comment]
[#49](https://github.com/rebase-network/hello-world/discussions/49) [#50](https://github.com/rebase-network/hello-world/discussions/50)
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsq0rtq2xlhdaettwrxyzgl22zm3wddktedsjkylql

**Comment by yyd818** [comment]
我正在学习nft知识，希望在这里和大家共同成长

**Comment by MingxiYe** [comment]
Link: #56 
wallet: ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqw4rc330sneva3d9yaztgzujak502ws43s3a290s

**Comment by lw1992** [comment]
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqvf6kq4uu3s7au0vasjt4qy7lhgukeaacgp5ulv7

**Comment by haoyh** [comment]
Link：[#66]
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqwffkecknm22lnjlzf4hywrmyclmqg3k7q2e9w3n

**Comment by 0x1be20** [comment]
https://github.com/rebase-network/hello-world/discussions/65
ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqd4j7e44yclfsxqq9upm8kakcdxwuxwafgvwxz7k

**Comment by NftTopBest** [comment]
Web3 NFT 浏览器
https://github.com/rebase-network/hello-world/discussions/3

ckb1qqypm0l63rdt2jayymfrrjnyadmqe630a8skwcdpmfqqmgdje0sjsqvcxfveauw2vqh6tt0qp8xfuxmjed469cq6nr65a

**Comment by luckyyang** [comment]
NFT 已经发送，各位可以在 https://rostra.xyz/mynfts 查看
![2022-06-17 at 16 53](https://user-images.githubusercontent.com/1963646/174263954-c7909d7b-3905-4a8b-9e3d-74cdc46b4f47.png)
